environs/tools/build_test.go

```diff
@@ -51,6 +51,11 @@ func (b *buildSuite) SetUpTest(c *gc.C) {
 	dir1 := c.MkDir()
 	dir2 := c.MkDir()
 
+	// Ensure we don't look in the real /usr/lib/juju for jujud-versions.yaml.
+	oldVersionFileFallback := tools.VersionFileFallbackDir
```

Comment:
There's a PatchValue that can be used here - does cleanup automatically

Reply:
Oh, nice - thanks.